### PR TITLE
Feature/suma form

### PR DESCRIPTION
**Pull request by arbulu89**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +50,30 @@ The script will overwrite any file with the same names).
 cd saphanabootstap-formula/examples
 sudo ./deploy.sh
 ```
+
+## Build
```
Comment (diegoakechi) [inline README.md]: Thanks for documenting this!

File: saphanabootstrap-formula.spec [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+%files
+%defattr(-,root,root,-)
+%license LICENSE
+%doc README.md
+/srv/salt/%{fname}
+
+%dir %attr(0755, root, salt) /srv/salt
+
+%changelog CHANGELOG.md
```
Comment (diegoakechi) [inline saphanabootstrap-formula.spec]: @arbulu89 As standard, we use a `<<package_name>>.changes` file for the change log on openSUSE/SUSE distributions:

https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)

Comment (arbulu89) [reply saphanabootstrap-formula.spec]: @diegoakechi yeah, initially I didn't even want to upload the .spec file, usually they are not uploaded. I will remove this file by now

**Comment by diegoakechi** [comment]
@arbulu89 I made a comment about the changelog, but can you please, check the conflicts too?

File: saphanabootstrap-formula.spec [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+%files
+%defattr(-,root,root,-)
+%license LICENSE
+%doc README.md
+/srv/salt/%{fname}
+
+%dir %attr(0755, root, salt) /srv/salt
+
+%changelog saphanabootstrap-formula.changes
```
Comment (diegoakechi) [inline saphanabootstrap-formula.spec]: The `%changelog` macro will assume this file as default, so you can change to:
```suggestion
%changelog
```

**Pull request merged**